src/Platform/Microsoft.Testing.Platform.MSBuild/buildMultiTargeting/Microsoft.Testing.Platform.MSBuild.props

```diff
@@ -1,10 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <Project>
-  <PropertyGroup>
-    <!-- This is the knob to disable completely for this project the machinery -->
-    <IsTestingPlatformApplication Condition=" '$(IsTestingPlatformApplication)' == '' ">True</IsTestingPlatformApplication>
```

Comment:
The idea of setting it that early was to say that if you use `MTP MSBuild` you are a test app.

Reply:
This doesn't look correct to me.
Is `IsTestingPlatformApplication` intended to be read by us only, and set by anyone? In that case, we know we read it late enough, and we can set the default late enough.

If it's intended to be read by others, then there is a design flaw here. It should only be read inside of a `Target` to allow it to be set as late as possible

Reply:
To me the idea is similar to `IsTestProject` except this is MTP

Reply:
It's more complicated for the case of MTP. Let's discuss offline